ledger/participant-integration-api/src/main/scala/platform/store/backend/common/CommonStorageBackend.scala

```diff
@@ -268,8 +268,9 @@ private[backend] trait CommonStorageBackend[DB_BATCH] extends StorageBackend[DB_
       |    ledger_offset <= {endInclusive} and
       |    parameters.ledger_end >= ledger_offset
       |  order by ledger_offset asc
-      |  limit {pageSize}
-      |  offset {queryOffset}""".stripMargin
+      |  offset {queryOffset} rows
```

Comment:
according to postgres specification `next` and `rows` are noise words...are they noise words in oracle as well? (if not, that would be a reason enough to have limit for postgres and this for oracle separately)
if they are noise words, can we omit them?

Reply:
this goes for all such constructs

Reply:
believe the noise words are needed for this to be portable across all the dbs